### DISCUSSION TITLE
FLUX and SD3

### DISCUSSION THREAD

**Discussion by TimmekHW** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#343] [Ideas]
Please add support for FLUX and SD3.
I use https://github.com/lllyasviel/stable-diffusion-webui-forge

![image](https://github.com/user-attachments/assets/41a01afd-4295-40bc-a621-8939dbc18e99)

![image](https://github.com/user-attachments/assets/9523f081-901b-4c8c-937c-056a26570e2b)

**Comment by Wontfallo** [comment]
Hey, I'm really confused about why this isn't a top priority for Nvidia. It's been a year, and it only works with automatic1111 webui and not consistently.

When it does work, it's incredible! Imagine generating 1024x1024 SDXL images in just 2.3 seconds at 80 steps. It's mind-blowing. So, what's the deal, Nvidia? Why aren't the developers of ComfyUI, Forge, and Fooocus supporting this? It should be a top priority; it's such a waste that it's not. This could be a game-changer for Flux, and it would save so much energy since generation times would be reduced by seven times.

Seriously, can someone explain why this hasn't become the standard and why it's not supported anywhere else? Nvidia RTX graphics cards are the best for inference and image generation, but they're not even close to reaching their full potential. Do people just not know about this? If that's the case, we can educate them, but what's Nvidia's excuse for not supporting this more?
